packages/liveblocks-core/src/lib/autoRetry.ts

```diff
@@ -46,3 +47,9 @@ export async function autoRetry<T>(
     await wait(delay);
   }
 }
+
+export class StopRetrying extends Error {
+  constructor(reason: string) {
+    super(reason);
+  }
+}
```

Comment:
Maybe we should DRY this up with the existing:

https://github.com/liveblocks/liveblocks/blob/699db03ac6d9fb684c439a9e1e087eef80e2b088/packages/liveblocks-core/src/connection.ts#L192-L200

…and just import that one from `core` now?

Reply:
Good idea! Not ideal to have it defined in multiple places even though we may plan to remove it in future.